README.md

```diff
@@ -1,68 +1,206 @@
+    0.4814468172064984,
+    0.9873832635128231
+]
+```
+
+#### `/flood-tiles/{zoom}/{x}/{y}.png`
+
+Used to get a rendered PNG tile at a given flood level for any given polygon. In addition to the `zoom`, `x`, and `y` path parameters, accepts a JSON blob as a POST parameter with keys of `multiPolygon`, `floodLevel`, `minElevation` and `maxElevation`, the last three of which all should be single float values. We must specify both minimum and maximum elevation here to allow the use of the proper color ramp which applies to the entire polygon, not just the elevation values within the tile being requested. Also, since the output is an image, we must specify an `Accept:image/png` header.
+
+```http
+$ http --download --output eastOmaha-12-956-1531.png :8090/flood-tiles/12/956/1531.png Accept:image/png multiPolygon:=@eastOmaha.json floodLevel:=10 minElevation:=293 maxElevation:=314
+HTTP/1.1 200 OK
+Content-Length: 4446
+Content-Type: image/png
+Date: Mon, 18 Jan 2016 19:16:52 GMT
+Server: spray-can/1.3.3
+
+Downloading 4.34 kB to "eastOmaha-12-956-1531.png"
+Done. 4.34 kB in 0.00062s (6.80 MB/s)
+```
+
+And the produced image:
+
+![eastOmaha-12-956-1531.png](https://cloud.githubusercontent.com/assets/1430060/12365387/7ec69ac6-bba2-11e5-8463-1724b24e739b.png)
+
+### Build
+
+To build the server JAR, execute the following:
 
 ```bash
-$ curl --silent \
-    --data "" \
-    "http://localhost:8090/jobs?sync=true&context=geoprocessing-context&appName=geoprocessing&classPath=org.azavea.usaceflood.geoprocessing.SummaryJob"
+$ ./sbt "project server" assembly
 ```
 
-#### Note:
+which should result in `server/target/scala-2.10/usaceflood-server-assembly-0.1.0-SNAPSHOT.jar`.
+
+### Usage
+
+The server expects three environment variables to be set:
+
+| Environment Variable Name | Default Value | Description |
+| ------------------------- | ------------- | ----------- |
+| `RDD_CATALOG_PATH`        | `/flood-data/catalog` | The path of `catalog` produced by the ingestion |
+| `DEM_10M_LAYER`           | `usace-mississippi-dem-10m-epsg4269` | The directory containing detailed 10m resolution tiles in EPSG:4269 |
+| `DEM_XYZ_LAYER`           | `usace-mississippi-dem-xyz` | The directory containing pyramided tiles in WebMercator |
+
+If running the server locally, if these variables are not found the defaults will be used. The provisioning of the VM does set these variables, but the default values are set to what they should be inside the VM.
 
-If you are using docker-machine to host the docker container, then be sure to use the correct IP address in the above curl statements:
+The actual server is run by executing the following:
 
 ```bash
-$ export DOCKER_IP=`docker-machine ip $DOCKER_MACHINE_NAME`
-$ curl --silent \
-    --data-binary @summary/target/scala-2.10/usaceflood-geoprocessing-assembly-0.0.1.jar \
-    "http://$DOCKER_IP:8090/jars/geoprocessing"
+$ spark-submit server/target/scala-2.10/usaceflood-server-assembly-0.1.0-SNAPSHOT.jar
```

Comment:
These look like instructions for running on your host. Consider mentioning where your data has to be for this to work.